components/03-modules/products-grid-item/_products-grid-item.scss

```diff
@@ -0,0 +1,194 @@
+        display: flex;
+        flex-flow: row nowrap;
+        justify-content: center;
+        align-items: flex-start;
+        width: 100%;
+        margin-bottom: $products-grid-item__image-margin-bottom;
+    }
+
+    &__name {
+        font-weight: $products-grid-item__name-font-weight;
+        color: $products-grid-item__name-color;
+        font-size: $products-grid-item__font-size-medium;
+        padding: $products-grid-item__text-padding;
+
+        @include mq($screen-m) {
+            font-size: $products-grid-item__font-size-large;
+        }
+    }
+
+    &__button {
+        &--active {
+            @include mq($screen-l) {
+                display: inline-flex;
+            }
+        }
+    }
+
+    &__price {
+        flex: 1 0 auto;
+        font-weight: $font-weight-bold;
+        text-align: right;
+        color: $products-grid-item__price-base-color;
+        padding: $products-grid-item__text-padding;
+        margin-top: 6px; // product name aligment
+    }
+
+    &__old-price {
+        @include mq($screen-m) {
+            display: block;
+            margin-right: 0;
+        }
+    }
+
+    &__special-price {
+        @include mq($screen-m) {
+            display: block;
+        }
+    }
+
+    &__extra {
+        opacity: 0;
+        height: 0;
+        .products-grid-item--active & {
+            opacity: 1;
+            height: auto;
+        }
+    }
+
+    &__options,
+    &__details {
+        margin-bottom: $products-grid-item__base-spacing;
+    }
+
+    &__actions {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        width: 100%;
+    }
+
+    &__action-icons {
+        display: flex;
+        align-items: center;
+        justify-content: flex-end;
+    }
+
+    &__action-item {
+        box-shadow: none;
+        background: transparent;
+        border: none;
+        width: $products-grid-item__action-item-size;
+        height: $products-grid-item__action-item-size;
+        margin-right: $products-grid-item__action-item-spacing;
+        &:hover {
+            cursor: pointer;
+        }
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+
+    &__action-icon {
+        width: $products-grid-item__action-icon-size;
+        height: $products-grid-item__action-icon-size;
+        &--wishlist {
+            width: $products-grid-item__action-icon-size--wishlist;
+            height: $products-grid-item__action-icon-size--wishlist;
+        }
+    }
+}
+
+$swatch-attribute-options__item-width: 32px !default;
+$swatch-attribute-options__small-rect-size: 32px !default;
+$swatch-attribute-options__image-selected-border: 1px solid $green !default;
+$swatch-attribute-options__item-spacing: $spacer !default;
+$swatch-attribute-options__underline-offset-bottom: $spacer !default;
+$swatch-attribute-options__link-bar-height: 8px !default;
+$swatch-attribute-options__link-bar-bg-color: $green !default;
+.swatch-attribute-options {
```

Comment:
It had to be a different file.

Reply:
sure. It was just here to ask if It's better to move it oustide or refactor to product-view namespace and leave it here.